x-pack/plugins/cases/server/client/cases/get.ts

```diff
@@ -75,20 +77,53 @@ export const getCaseIDsByAlertID = async (
       Operations.getCaseIDsByAlertID.savedObjectType
     );
 
+    // This will likely only return one comment saved object, the response aggregation will contain
```

Comment:
>This will likely only return one comment saved object
What will be the response if the same alert is attached to multiple cases?

Reply:
The service requests `page: 1, perPage: 1` so I think it'll only ever return 1 comment. But the aggregations of that single comment will have all the unique case IDs that we need.